app/views/rules_of_origin/steps/_sidebar_section.html.erb

```diff
@@ -20,6 +20,20 @@
   </div>
 
   <div class="app-step-nav__panel js-panel js-hidden" id="step-panel-check-youre-allowed-to-drive-1">
+    <% section_content = t("rules_of_origin.steps.sections.#{sidebar_section.first}_info_html",
+                           commodity_code: current_step.commodity_code,
+                           trade_country: current_step.trade_country_name,
+                           commodity_link: link_to(current_step.commodity_code,
+                                                   commodity_path(current_step.commodity_code,
+                                                                  country: current_step.trade_country_code)),
+                           default: nil) %>
```

Comment:
what do you think, to avoid assignment and pollution in the view, to assign this in the controller (maybe even a helper?) 🤔